tests/std/tests/P0896R4_views_elements/test.cpp

```diff
@@ -414,4 +414,13 @@ int main() {
         STATIC_ASSERT(test_one(s));
         test_one(s);
     }
+
+    { // Validate a view borrowed range
+
+        constexpr auto v = views::iota(0ull, ranges::size(expected_keys)) | std::views::transform([](auto i) {
+            return std::make_pair(expected_keys[i], expected_values[i]);

```

Comment:
This test is already `using namespace std` so `std::views::transform` and `std::make_pair` don't need to say `std`.

Reply:
```suggestion
        constexpr auto v = views::iota(0ull, ranges::size(expected_keys)) | views::transform([](auto i) {
            return make_pair(expected_keys[i], expected_values[i]);
```